### PR TITLE
Deprecate AlternatingSessionUserList in favor of added SessionTabOverhaul release

### PR DESCRIPTION
**Pull request by Banane9**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -4219,6 +4222,39 @@
...
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/Banane9/NeosSessionTabOverhaul/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosSessionTabOverhaul/releases/download/v1.0.0/SessionTabOverhaul.dll",
+                            "sha256": "299B3B0AED8B3F0BFB4447CCAC2203303DF00DC431851D5D2CE42E8C5E9662FF"
```
Comment (XDelta) [inline manifest.json]: Hash doesn't match provided dll

File: manifest.json [hunk 2]
```diff
@@ -4219,6 +4222,39 @@
...
+                            "version": "*"
+                        }
+                    }
+                }
+            }
+        },
+        "Banane9.SessionTabOverhaul": {
+            "name": "SessionTabOverhaul",
+            "description": "Overhauls the Session Tab of the Dash, most notably the Users list, by adding more details and options. Screenshots on the Github page. Replaces Banane9.AlternatingSessionUserList.",
+            "category": "General UI Tweaks",
```
Comment (XDelta) [inline manifest.json]: "Dash Tweaks" Seems to be a more fitting category

**Review by XDelta — CHANGES_REQUESTED** [review]

**Comment by XDelta** [comment]
hm, github magic I guess lol

File: manifest.json [hunk 3]
```diff
@@ -4382,7 +4390,43 @@
...
+                "Banane9": {
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/Banane9/NeosSessionTabOverhaul/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosSessionTabOverhaul/releases/download/v1.0.0/SessionTabOverhaul.dll",
```
Comment (XDelta) [inline manifest.json:4418]: Non-blocking: AssemblyCopyright set to 2021

**Review by XDelta — APPROVED** [review]

**Comment by Banane9** [comment]
Thanks, GitHub 🤪

**Pull request merged**